hack/build-cross.sh

```diff
@@ -35,6 +35,7 @@ OS_GOFLAGS="${OS_GOFLAGS:-} ${OS_IMAGE_COMPILE_GOFLAGS}" os::build::build_static
 OS_RELEASE_ARCHIVE="openshift-origin"
 OS_BUILD_PLATFORMS=("${platforms[@]}")
 os::build::place_bins "${OS_CROSS_COMPILE_BINARIES[@]}"
+echo "${OS_GIT_COMMIT}" > "${OS_LOCAL_RELEASEPATH}/.commit"
```

Comment:
Not really (not without a lot of complexity).  At minimum you can't do this in a dirty state today so writing out a message and changing the commit to be whatever the tree state flag is (there was COMMIT-dirty, that could work).